core/webpack.config.js

```diff
@@ -81,7 +81,11 @@ module.exports = {
     new ExtractTextPlugin('luigi.css'),
     new CopyWebpackPlugin([
       'node_modules/oidc-client/dist/oidc-client.min.js',
-      'node_modules/fundamental-ui/dist/SAP-icons.woff'
+      'node_modules/fundamental-ui/dist/SAP-icons.woff',
+      {
+        from: 'node_modules/fundamental-ui/dist',
```

Comment:
Maybe we only need to import fundamental-ui.min.scss? Futhermore, we could also add SAP-icons.woff to `fundamental-ui` folder (and update `static.css` accordingly)

Reply:
Done